tests/test_statsummary.py

```diff
@@ -0,0 +1,35 @@
+from CovidTracker.calculate_stat_summary import calculate_stat_summary
+from CovidTracker.get_covid_data import get_covid_data
+import pandas as pd
+import pytest
+
+
+def test_calculate_stat_summary():
+    """Test the calculate_stat_summary() function"""
+
+    input = get_covid_data()
+   
+    # Test output type
+    assert isinstance(calculate_stat_summary(input), pd.DataFrame)
+    
+    # Test output size
+    assert calculate_stat_summary(input).shape == (14, 4)
+
+    # Test output column names
+    assert 'date_report' in calculate_stat_summary(input).columns
```

Comment:
input df will not necessarily have these columns  based on what data_type was used in get_covid_data()

Reply:
In the test example, the df dataframe exactly contains the date_report column.